src/test/opentrackIO_parser/python/opentrackIO_parser.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+#
+# opentrackIO_parser.py
+#
+# Reference code for parsing/decoding opentrackIO samples
+#
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the SMTPE RIS OSVP Metadata Project
+#
+# Example run: python3 opentrackIO_parser.py --file=opentrackio_complete_static_example-20241001.json --schema=opentrackio_schema_20241001.json
+# This is tested against the generated "complete_static_example" and "complete_dynamic_example" json
+
+import os
+import argparse
+from opentrackIOlib import *
+
+parser = argparse.ArgumentParser(description='OpenTrackingProtocol parser')
+parser.add_argument('-f', '--file', help='The JSON input file.', default=None)
+parser.add_argument('-s', '--schema', help='The schema (JSON) input file.', default=None)
+parser.add_argument('-v', '--verbose', help='Make script more verbose',action='store_true')
+args = parser.parse_args()
+
+sample_text = ''
+schematext = ''
+verbose = False
+
+if (args.schema):
+    if os.path.exists(args.schema):
+        schemapath = args.schema
+        if (schemapath != None):        
+            with open(schemapath, 'r') as fd:
+                print("Reading OTIO schema file: {0}".format(schemapath))
+                lines = fd.readlines()
+                for line in lines:
+                    schematext = schematext + line
+if (args.file):
+    if os.path.exists(args.file):
+        filepath = args.file
+        if (filepath != None):        
+            with open(filepath, 'r') as fd:
+                print("Reading OTIO sample file: {0}".format(filepath))
+                lines = fd.readlines()
+                for line in lines:
+                    sample_text = sample_text + line
+if (args.verbose):
+    verbose = True
+
+sample = OpenTrackIOProtocol(sample_text,schematext,verbose) # a "Sample" is a de-serialized JSON object containing the protocol
+sample.Parse()                # parse the actual JSON of the protocol
```

Comment:
Method names should be lower-case underscore separated to match the rest of CamDKit. Class name is correct (capitalized camel-case)

Reply:
Done